lavamoat/browserify/beta/policy.json

```diff
@@ -230,165 +235,194 @@
         "webpack>events": true
       }
     },
-    "@ethereumjs/tx>@ethereumjs/util": {
+    "@keystonehq/metamask-airgapped-keyring>@ethereumjs/tx>@ethereumjs/util": {
       "globals": {
-        "console.warn": true,
-        "fetch": true
+        "console.warn": true
       },
       "packages": {
-        "@ethereumjs/tx>@ethereumjs/rlp": true,
+        "@keystonehq/metamask-airgapped-keyring>@ethereumjs/tx>@ethereumjs/rlp": true,
+        "browserify>buffer": true,
         "@ethereumjs/tx>ethereum-cryptography": true,
-        "webpack>events": true
+        "webpack>events": true,
+        "browserify>insert-module-globals>is-buffer": true,
+        "eth-lattice-keyring>@ethereumjs/util>micro-ftch": true
```

Comment:
For the record: We were already using `micro-ftch` in some other policy entries.